spark-indexer/src/main/java/org/uniprot/store/spark/indexer/main/experimental/IndexDataStoreMainMultiThread.java

```diff
@@ -25,12 +25,13 @@
 public class IndexDataStoreMainMultiThread {
 
     public static void main(String[] args) {
-        if (args == null || args.length != 3) {
+        if (args == null || args.length != 4) {
             throw new IllegalArgumentException(
                     "Invalid arguments. Expected "
                             + "args[0]= release name"
                             + "args[1]= collection names (for example: uniprot,uniparc,uniref)"
-                            + "args[2]=spark master node url (e.g. spark://hl-codon-102-02.ebi.ac.uk:37550)");
+                            + "args[2]=spark master node url (e.g. spark://hl-codon-102-02.ebi.ac.uk:37550)"
+                            + "args[3]= taxonomy db (e.g.read or fly)");
```

Comment:
Can you please add an extra validation here to make sure that the value is  read or fly?